services/web/server/src/simcore_service_webserver/invitations/_client.py

```diff
@@ -79,6 +126,7 @@ async def ping(self) -> bool:
     # service API
     #
 
+    @_handle_exceptions_as_invitations_errors
     async def extract_invitation(
```

Comment:
Q: would it make sense to have some retry mechanism somewhere as well? or is this already there somewhere?

Reply:
this is not heavy use but it might indeed have a retry